Neos.Neos/Migrations/Mysql/Version20230307111325.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neos\Flow\Persistence\Doctrine\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Drop obsolete tables
+ */
+final class Version20230307111325 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->abortIf(
+            !$this->connection->getDatabasePlatform() instanceof \Doctrine\DBAL\Platforms\MariaDb1027Platform,
+            "Migration can only be executed safely on '\Doctrine\DBAL\Platforms\MariaDb1027Platform'."
+        );
+
+        $this->addSql('DROP TABLE neos_neos_projection_asset_usage');
```

Comment:
Should we really drop this? I found the class `AssetUsageRepository` which uses this table. 🤔

Besides: The class would suggest that this migration should reside inside `Neos.ESCR.AssetUsage`.

Reply:
You are right